frontend/www/js/omegaup/components/arena/ClarificationList.vue

```diff
@@ -123,6 +118,13 @@ export default class ArenaClarificationList extends Vue {
   T = T;
   problemAlias = this.problems[0]?.alias ?? null;
   message: null | string = null;
+
+  onNewClarification(): void {
+    if (this.problemAlias == null || this.message == null) return;
+    const problemAlias = this.problemAlias;
+    const message = this.message;
+    this.$emit('new-clarification', { problemAlias, message });
```

Comment:
nit:

```suggestion
    this.$emit('new-clarification', { problemAlias: this.problemAlias, message: this.message });
```

esto es un poquito más corto.